nursery/manipulate-user-privileges.yml

```diff
@@ -0,0 +1,10 @@
+rule:
+  meta:
+    name: manipulate user privileges
```

Comment:
add this to `modify access privileges`?

Reply:
`modify access privileges` appears specific to a process whereas `LsaAddAccountRights` is specific to a user which is why I kept them separate.  Thoughts?

Reply:
ah, right, never mind